packages/touchegg-git/touchegg-git.pacscript

```diff
@@ -0,0 +1,30 @@
+name="touchegg-git"
+pkgname="touchegg"
+url="https://github.com/JoseExposito/touchegg.git"
+build_depends="git build-essential gdb cmake debhelper libudev-dev libinput-dev libpugixml-dev libcairo2-dev libx11-dev libxtst-dev libxrandr-dev libxi-dev libglib2.0-dev libgtk-3-dev"
+breaks="${pkgname} ${pkgname}-bin ${pkgname}-deb ${pkgname}-app"
+description="Touchégg is an app that runs in the background and transform the gestures you make on your touchpad or touchscreen into visible actions in your desktop."
+maintainer="Zahrun <Zahrun@github.com>"
+pkgver() {
+  git ls-remote "${url}" refs/heads/master | cut -f1 | cut -c1-8
+}
+version="$(pkgver)"
+
+prepare() {
```

Comment:
You can remove this function, as it’s not used

Reply:
Great, thanks